scene/gui/popup_menu.cpp

```diff
@@ -282,14 +282,15 @@ void PopupMenu::_gui_input(const Ref<InputEvent> &p_event) {
 	}
 
 	// Make an area which does not include v scrollbar, so that items are not activated when dragging scrollbar.
-	Rect2 item_clickable_area = control->get_global_rect();
-	float scroll_width = scroll_container->get_v_scrollbar()->is_visible_in_tree() ? scroll_container->get_v_scrollbar()->get_size().width : 0;
-	item_clickable_area.set_size(Size2(item_clickable_area.size.width - scroll_width, item_clickable_area.size.height));
+	Rect2 item_clickable_area = get_visible_rect();
```

Comment:
Option a) change `get_visible_rect()` to `scroll_container->get_rect()`. The size of the scroll container will account for the offset of the margin container, while `get_visible_rect()` gets the rect of the entire popup.